### ISSUE TITLE
Aesthetic issue for double ci view

### ISSUE THREAD

**Issue by sammo3182** [fsolt/SWIIDweb#5]
When more countries are selected, the double CI view does not look very clear.

**Comment by fsolt** [comment]
I think that was because of the problem with the color of the CI, but I don't trust my color vision enough to be sure.  Is it resolved with the latest commit?

**Comment by sammo3182** [comment]
Not really. One cannot see there are two CIs in the new version. Suggest roll back to the way of 4677798a36919697781ae8e8fa666515f2be8926, although the legend would still be a problem. @fsolt

**Comment by fsolt** [comment]
Rolling back isn't the right thing to do, I'm afraid.  The old way was making the CI arbitrarily colored: rather than the specified `grey70`, it was assigning a color to the constant string "grey70".  Maybe easier to view, but not what was intended.

I think a solution may be to use a custom palette of lighter colors (or add alpha to the ribbon), and make the overlaid CI a darker shade of that same palette.  This shouldn't be a problem since the app only supports 4 series at a time; there's no need for the automatically generated ggplot infinite range of arbitrary colors.

**Comment by sammo3182** [comment]
@fsolt, yes. I also thought to change the alpha. The output looks better now. Have a check and let me know if you want me to modify anything or I can merge the branch into the master.

**Comment by fsolt** [comment]
It looks like the CIs are reversed, Hu: the smaller inner CI should be darker than the outer one, but that's not what we're getting at the moment.

**Comment by sammo3182** [comment]
I think I fixed it at this time. Have a check, @fsolt.
